doc/user/content/manage/access-control/rbac-terraform-tutorial.md

```diff
@@ -0,0 +1,377 @@
+---
+title: "Tutorial: Manage privileges with Terraform"
+description: "Add users, create roles, and assign privileges in Materialize with Terraform"
+menu:
+  main:
+    parent: access-control
+    weight: 25
+---
+
+{{< private-preview />}}
+
+This tutorial walks you through managing roles in Materialize with [Terraform](https://www.terraform.io/). By the end of this tutorial you will:
+
+* Create two new roles in your Materialize
+* Apply privileges to the new roles
+* Assign a role to a user
+* Modify and remove privileges on roles
+
+In this scenario, you are an DevOps engineer responsible for managing your Materialize account with code. You recently hired a new developer who needs privileges in a non-production cluster. You will create specific privileges for the new role that align with your business needs and restrict the developer role from having access to your production cluster.
+
+## Before you begin
+
+* Make sure you have a [Materialize account](https://materialize.com/register/?utm_campaign=General&utm_source=documentation) and already have a password to connect with.
+
+* You should be familiar with setting up a [Terraform project in Materialize](https://materialize.com/docs/manage/terraform/).
+
+* Have an understanding of permissions in Materialize. These grant resources in this tutorial are modeled after the [access-control tutorial](https://materialize.com/docs/manage/access-control/rbac-tutorial/).
+
+## Step 1. Create Role
+
+1. A role is a collection of privileges you can apply to users. First we will create a role resource in Terraform.
+
+    ```hcl
+    resource "materialize_role" "dev_role" {
+      name = "dev_role"
+    }
+    ```
+
+2. We will run Terraform to create this role.
+
+    ```shell
+    terraform apply
+    ```
+
+    {{< note >}}
+    All of the resources in this tutorial can be run with a single `terraform apply` but we will add and apply resources incrementally to better illustrate grants.
```

Comment:
Don't know why (it is a bug) but removing one tab here formats the text correctly and not as if it was code:

```
    {{< note >}}
  All of the resources in this tutorial can be run with a single terraform apply but we will add and apply resources incrementally to better illustrate grants.
    {{</ note >}}
```